extensions/ql-vscode/docs/test-plan.md

```diff
@@ -278,3 +281,40 @@ This requires running a MRVA query and seeing the results view.
 #### Test case 4: When variant analysis is in "failed" or "canceled" state
 1. Can view logs
 1. Results for finished queries are still downloaded.
+
+### MRVA repositories panel
+
+- [ ] Add a list
```

Comment:
Sorry to be a pain as the test plan items look great, but should they written in a style to match the rest? Not that I'm arguing that the existing style is 100% perfect, but it would be good for things to match.

Reply:
I'd welcome looking at changing the existing style. I agree writing out all the numbers is not the best for maintenance or reading. Perhaps open an issue so we don't forget / can discuss options.